solr/solrj/src/java/org/apache/solr/client/solrj/impl/CloudHttp2SolrClient.java

```diff
@@ -151,9 +152,9 @@ public static class Builder {
     protected Http2SolrClient.Builder internalClientBuilder;
     private RequestWriter requestWriter;
     private ResponseParser responseParser;
-    private long retryExpiryTime =
+    private long retryExpiryTimeNano =
         TimeUnit.NANOSECONDS.convert(3, TimeUnit.SECONDS); // 3 seconds or 3 million nanos
-    private int timeToLiveSeconds = 60;
+    private long timeToLiveSeconds = TimeUnit.SECONDS.convert(60, TimeUnit.SECONDS);
```

Comment:
LOL why?

Reply:
mostly to always use the same  pattern...   Though the more I look at it...   The funnier maybe it is...   Is there a pattern like "TimeUnit.SECONDS(60)"?

Reply:
It was fine the way it was; it reads in an obvious way.